src/network/networkd-sysctl.c

```diff
@@ -179,6 +179,21 @@ static int link_set_ipv6_hop_limit(Link *link) {
         return sysctl_write_ip_property_int(AF_INET6, link->ifname, "hop_limit", link->network->ipv6_hop_limit);
 }
 
+static int link_set_ipv6_retrans_time(Link *link) {
+        uint32_t retrans_time_ms;
+
+        assert(link);
+
+        if (!link_is_configured_for_family(link, AF_INET6))
+                return 0;
+
+        if (link->network->ipv6_retrans_time <= 0)
```

Comment:
Please use `timestamp_is_set()`, to refuse USEC_INFINITY.

Reply:
Done.